docker-compose.prod.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  firefly-app:
+    container_name: firefly-iii
```

Comment:
Is this needed? Even when switching from on env to another in the same folder, docker-compose will recreate the container automatically if needed.